### DISCUSSION TITLE
Plea to restore Hardcore killed by pvp character

### DISCUSSION THREAD

**Discussion by Cavebunny** [chromiecraft/chromiecraft#7257] [General]
![WoWScrnShot_080424_163824](https://github.com/user-attachments/assets/07bd7328-a0c6-4b29-9749-1db5093b7ea9)
![WoWScrnShot_080424_163444](https://github.com/user-attachments/assets/40f6c432-83ea-4477-8898-a2db50296ff4)
![WoWScrnShot_080424_163243](https://github.com/user-attachments/assets/62f07901-468a-485d-85c7-1579a19ecbeb)
I landed on elemental plateau. I saw no other player there. I attacked one water elemental. My "druid trees" were out. As I was in battle Yamaguchi came from behind and landed, pvp enabled, next to the battle. I tried to move away from the battle, but as the elemental died my "trees" got me pvp enabled and Yamaguchi attacked me. I tried to run further and heal, but he followed and killed me.
I've asked before, as a hunter, if pvp enabled players can trick me into pvp and was told it could not happen. This is quite upsetting.
Tekuho *is* about ten thousand xp short of L70.

**Comment by Amrx2** [comment]
Hi Cavebunny, I can confirm the character has been restored. Best of luck on the rest of your journey to 70!

**Reply by Cavebunny** [reply]
Many thanks!

I've dedicated much of my gaming experience as hardcore, particularly with Tekuho, to help other hardcore players, so it's very gratifying to see that I can continue with that mission.
